extensions/kafka/src/main/java/io/deephaven/kafka/ingest/KafkaStreamPublisher.java

```diff
@@ -127,7 +137,10 @@ public static ConsumerRecordToStreamPublisherAdapter make(
                 simpleKeyColumnIndex,
                 simpleValueColumnIndex,
                 parameters.getKeyToChunkObjectMapper(),
-                parameters.getValueToChunkObjectMapper());
+                parameters.getValueToChunkObjectMapper(),
+                parameters.getReceiveTimeColumnIndex(),
+                parameters.getKeyBytesColumnIndex(),
+                parameters.getValueBytesColumnIndex());
```

Comment:
More a note for Ryan and myself - wonder if we should just pass parameters into KafkaStreamPublisher instead of extracting all the values here.